backend/src/main/java/park_su_park/backend/service/UserService.java

```diff
@@ -0,0 +1,25 @@
+package park_su_park.backend.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import park_su_park.backend.domain.User;
+import park_su_park.backend.request_dto.RequestUserDto;
+import park_su_park.backend.repository.UserRepository;
+import park_su_park.backend.response_dto.ResponseUserDto;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class UserService {
+    private final UserRepository userRepository;
+
+    @Transactional
+    public ResponseUserDto save(RequestUserDto requestUserDto) {
+        User user = User.createUser(requestUserDto);
+        userRepository.save(user);
+        ResponseUserDto responseUserDto = ResponseUserDto.createResponseUserDto(user.getId(), user.getCreate_time(), requestUserDto);
+        return responseUserDto;
```

Comment:
이 부분은 그냥 읽어보고 넘기셔도 됩니다.

위의 코드가 정상 작동한다고 하더라도  

`작동하는 코드`인것은 맞지만  
제 스타일의 코드와는 거리가 좀 있는 것 같습니다.


제 스타일의 코드는 아래와 같습니다.
```suggestion
    public ResponseUserDto save(RequestUserDto requestUserDto) {
        User user = User.createUser(requestUserDto);
        User savedUser = userRepository.save(user);

        return ResponseUserDto.of(savedUser); // of는 직접 만들어야합니다.
```